singer-io/tap-recharge
# Add tap tester

# Description of change
- adding tap-tester tests as per [TDL-14836](https://jira.talendforge.org/browse/TDL-14836)
- this PR also includes commits from #13 

# Manual QA steps
 - ran tap manually
 - ran `singer-check-tap`
 - ran `target-stitch --dry-run`
 - ran tap-tester tests locally
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
